examples/xmllib/main.py

```diff
@@ -0,0 +1,77 @@
+# noqa: INP001 (ignore add __init__.py)
+
+
+from pathlib import Path
+
+import pandas as pd
+
+from dsp_tools import xmllib
+
+data_folder = Path("examples/xmllib/data")
+
+
+def _make_books(res_collection: xmllib.ResourceCollection) -> None:
+    df = pd.read_csv(data_folder / "Book.csv")
+    for _, row in df.iterrows():
+        one_book = _make_one_book(row)
+        res_collection.with_Resource(one_book)
+
+
+def _make_one_book(row: pd.Series) -> xmllib.Resource:
+    res = xmllib.Resource(res_id=row["id"], restype=":Book", label=row["label"])
+    titles = row["hasTitle"].split("||")
+    titles = [x.strip() for x in titles]
+    res.with_SeveralSimpleTexts(
+        prop_name=":hasTitle",
+        values=titles,
+    ).with_IntValue(
+        prop_name=":hasNumberOfPages",
+        value=row["hasNumberOfPages"],
+    ).with_SimpleText(prop_name=":hasAuthor", value=row["hasAuthor"])
```

Comment:
cosmetics: IMO this is far more readable like this:

```suggestion
    res.with_SeveralSimpleTexts(prop_name=":hasTitle", values=titles)
        .with_IntValue(prop_name=":hasNumberOfPages", value=row["hasNumberOfPages"])
        .with_SimpleText(prop_name=":hasAuthor", value=row["hasAuthor"])
```
(same goes for other cases below)

Reply:
I agree but when I ran ruff it was either this or something really weird...